src/App.js

```diff
@@ -1,5 +1,25 @@
+const BridgeGame = require("./controller/BridgeGame.js");
+const BridgeGameService = require("./service/BridgeGameService.js");
+const BridgeGameModel = require("./model/BridgeGameModel.js");
+const InputView = require("./view/InputView.js");
+const OutputView = require("./view/OutputView.js");
+
 class App {
-  play() {}
+  #bridgeGameService;
+  #bridgeGameController;
+  constructor() {
```

Comment:
airbnb에도 있는지는 모르지만 [eslint lines-between-class-members](https://eslint.org/docs/latest/rules/lines-between-class-members)에 의하면 아래처럼 한 줄 띄고 쓰는 것이 좋은 것 같아요!
```suggestion
  #bridgeGameService;
  
  #bridgeGameController;
  
  constructor() {
```

Reply:
필드도 한 줄 띄고써야하는 컨벤션이 있었군요!

린트를 키고 작성을 하지 않다보니 놓쳤던 디테일들이 꽤 있네요
역시 린트를 켜둬야하는 습관을 들여야.. ㅋㅋㅋ